lib/index.js

```diff
@@ -32,9 +28,9 @@ module.exports = (opts = {}) => {
         }
 
         try {
-            if(isFunction(secret)) {
-              // secret is RSA callback (e.g. jwks-rsa.expressJwtSecret)
-              secret = await getRSASecret(ctx, secret, token);
+            if(typeof secret === 'function') {
+              // secret is RSA provider
```

Comment:
I don't think we even need a comment here. Could we name the function getSecret? This interface may be used contexts other than JWKS.